platform/.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
+  "recommendations": [
+    "esbenp.prettier-vscode",
+    "dbaeumer.vscode-eslint",
+    "bradlc.vscode-tailwindcss",
+    "github.vscode-pull-request-github",
+    "firsttris.vscode-jest-runner",
+    "christian-kohler.npm-intellisense",
+    "wakatime.vscode-wakatime",
+    "willstakayama.vscode-nextjs-snippets",
+    "dsznajder.es7-react-js-snippets"
+  ]
+}
```

Comment:
Curious about these configurations. Do we need them or they sneaked into the PR by mistake? @FaithDaka

Reply:
They are required VS Code extensions that maximise efficiency of a=the dev team. m=Makes it easy for new collaborators to plug in. Especially the snippets and intellisense

Reply:
Alright thanks